core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -875,6 +878,82 @@ public static String translateWithCharset(String s, String transcodingName) {
     }
   }
 
+  /** SQL {@code PARSE_URL(urlStr, partToExtract [, keyToExtract] )} function. */
```

Comment:
Looks like the Java Doc is more suitable for placing on the method below.

Reply:
I have adjusted the places of these two method, and the doc.